lastminute.yml

```
@@ -11,6 +11,27 @@
     - role: p100
 
   tasks:
+    - name: switch from files to compat in nsswitch
+      tags: nsswitch
+      ansible.builtin.replace:
+        path: /etc/nsswitch.conf
+        regexp: '^{{item}}(.*)files'
```

```suggestion
        regexp: '^{{ item }}(.*)files'
```

I think the linter should flag this